tests/system/common.py

```diff
@@ -785,3 +786,15 @@ def task_by_name(tasks, name):
 
 async def assert_event(event_type, event_stream, within=10):
     await asyncio.wait_for(find_event(event_type, event_stream), within)
+
+
+@contextlib.contextmanager
+def docker_ipv6_network(network_name):
```

Comment:
definitely should have `_fixture` in the name (e.g. `docker_ipv6_network_fixture`)